### PR TITLE
Pull request template

### PR DESCRIPTION
**Pull request by prawl**
Now auto generating the pull_request_template.md file and the .github directory when setting up boxcar

### CODE REVIEW

File: lib/boxcar/app_builder.rb [hunk 1]
```diff
@@ -81,6 +81,11 @@ def install_devise_token_auth
       copy_file "user.rb", "app/model/user.rb"
     end
 
+    def create_github_markdown
+      Dir.mkdir(".github/")
```
Comment (dkniffin) [inline lib/boxcar/app_builder.rb]: I don't think you need this line. I'm pretty sure `copy_file` will create the directory for you, but I'm not 100% sure about that.

**Review by dkniffin — APPROVED** [review]
One comment, but otherwise 👍 

Edit: I actually noticed a few more. It doesn't look like there's a way to revoke an approval, but I'd like to see those made before merging this.

File: lib/boxcar/commands/new.rb [hunk 2]
```diff
@@ -45,6 +45,11 @@ def setup_secrets
         build :create_secrets_example
       end
 
+      def create_github_template
```
Comment (dkniffin) [inline lib/boxcar/commands/new.rb]: Oh, one more comment. This should be named `setup_github_template`, and you'll need an `invoke :setup_github_template` in `boxcar_customization`. Something about that should probably be added to the comment at the top of the file as well, if you want to do that.

**Comment by dkniffin** [comment]
Another thing: we'll want a test for this. [Something like this](https://github.com/smashingboxes/boxcar/blob/master/spec/support/shared_examples/basic_setup_steps.rb#L26-L28) should be sufficient

File: lib/boxcar/app_builder.rb (hunk 1 above)
Comment (prawl) [reply lib/boxcar/app_builder.rb]: It looks like you're correct!  Removing this line.

**Comment by prawl** [comment]
@dkniffin Let me know if that addresses your comments.

**Review by dkniffin — APPROVED** [review]
Yep, that looks good 👍

**Pull request merged**